compiler/testData/diagnostics/tests/when/ExhaustiveValOverConditionalInit.fir.kt

```diff
@@ -4,8 +4,8 @@ fun foo(a: Boolean, b: Boolean): Int {
         x = 1
     }
     when (b) {
-        true -> x = 2
-        false -> x = 3
+        true -> <!VAL_REASSIGNMENT!>x<!> = 2
+        false -> <!VAL_REASSIGNMENT!>x<!> = 3
```

Comment:
This 2nd error raises the question on how often we want to report errors. It seems like old FE reported an error only once for the above `true` case.

Reply:
I think yet we can report both of them, it's not a major problem.